NuRadioMC/SignalProp/directRayTracing.py

```diff
@@ -48,13 +48,13 @@ def get_path_length(self, iS):
     def get_travel_time(self, iS):
         traveltime = 0
         path = self.get_path(iS)
-        segment = [path[0][1]-path[0][0],path[1][1]-path[1][0],path[2][1]-path[2][0]]
+        segment = [path[1][0]-path[0][0],path[1][1]-path[0][1],path[1][2]-path[0][2]]
```

Comment:
Just to clarify, what are the indices of `path`?

Reply:
The function `get_path` makes a straight line from the shower to the observer and splits it into 1000 segments (points). The dimension of the returned `path` is [n_points,n_dimensions]. So in this line, we calculate the vector representing a segment (first point minus zeroth point).

Reply:
It would be better just to write `segment = path[1,:]-path[0,:]`, but I tried to keep the changes minimal to focus on the bugfix.